tasks/libs/copyright.py

```diff
@@ -22,17 +22,26 @@
     r'^// Copyright 20[1-3][0-9]-([Pp]resent|20[1-3][0-9]) Datadog, (Inc|Inmetrics)\.$',
 ]
 
-# These path patterns are excluded from checks
 PATH_EXCLUSION_REGEX = [
     # These are auto-generated files but without headers to indicate it
+    '/pkg/remoteconfig/client/products/apmsampling/.*_gen(_test){,1}.go',
+    '/pkg/proto/msgpgo/.*_gen(_test){,1}.go',
     '/pkg/security/probe/accessors.go',
+    '/pkg/security/probe/activity_dump_gen_linux.go',
     '/pkg/security/probe/custom_events_easyjson.go',
+    '/pkg/security/probe/fields_resolver.go',
     '/pkg/security/probe/serializers_easyjson.go',
+    '/pkg/security/secl/model/.*_gen(_test){,1}.go',
     '/pkg/security/secl/model/accessors.go',
+    '/pkg/trace/pb/.*_gen(_test){,1}.go',
     # These are files that we should not add our copyright to
-    '/third_party/golang/',
-    '/third_party/kubernetes/',
+    '/internal/patch/grpc-go-insecure/',
+    '/internal/patch/logr/funcr/funcr(_test){,1}.go',
+    '/internal/patch/logr/funcr/internal/logr/',
+    '/internal/third_party/golang/',
+    '/internal/third_party/kubernetes/',
 ]
+# These path patterns are excluded from checks
```

Comment:
I think this comment made more sense before the list 😄

Reply:
Doh!  Not sure how that ended up there 😅